Makefile

```diff
@@ -0,0 +1,38 @@
+GO := go
+
+GO_BUILD_PACKAGES := ./cmd/...
+GO_BUILD_BINDIR :=./bin
+
+all: clean vendor test-unit build
```

Comment:
always clean? I tend to set default targets to do build-over for development speed.
It has its downsides, but nothing that `make clean` can't solve :)

Reply:
Fair :). I can move this down in the list so it's not the default target.